src/security/authentication.ts

```diff
@@ -1,13 +1,10 @@
 import * as express from 'express';
-import * as NodeCache from 'node-cache';
 
 import { AuthenticatedRequest } from '../models';
 import { checkReporterKey } from '../data';
+import { Cache, logger } from '../core';
 
-const reportersCache = new NodeCache({
-  stdTTL: 60 * 60 * 2, // Every 2 hours reread key from DB.
-  checkperiod: 60 * 30 // Clear old cache every 30 minutes.
-});
+const reportersAuthCache = new Cache(2 * 60 * 60);
```

Comment:
Cache for this one isn't necessary, the report route wouldn't be called so frequently.

Reply:
I removed it in 8ae4d3da051b02ae74eaf205572a24405a8b6cef, but it was fun.